soh/src/code/z_en_item00.c

```diff
@@ -416,8 +428,14 @@ void EnItem00_Init(Actor* thisx, PlayState* play) {
             break;
         case ITEM00_TUNIC_ZORA:
         case ITEM00_TUNIC_GORON:
-            this->actor.objBankIndex = Object_GetIndex(&play->objectCtx, OBJECT_GI_CLOTHES);
-            Actor_SetObjectDependency(play, &this->actor);
+            // Remove object dependency for Enemy Randomizer and Crowd Control to allow Like-likes to
+            // drop equipment correctly in rooms where Like-likes normally don't spawn.
+            if (CVar_GetS32("gRandomizedEnemies", 0) || CVar_GetS32("gCrowdControl", 0)) {
+                this->actor.objBankIndex = 0;
+            } else {
+                this->actor.objBankIndex = Object_GetIndex(&play->objectCtx, OBJECT_GI_CLOTHES);
+                Actor_SetObjectDependency(play, &this->actor);
+            }
```

Comment:
would be nice to have this not be copypasta but i can't think of a better solution at the moment

Reply:
I take it you're talking about this being the same for every like-like drop. Technically it could be pulled into its own function instead, but ideally this would all be removed and replaced with `this->actor.objBankIndex = 0;` anyway once we properly remove object dependency. I'll leave it up to you if you think it's worth doing the extra function for now.

Reply:
it's probably fine as is, but having it in a function would be nice if something needs to change in the future (always nicer to only have to change things in one place)

Reply:
Doesn't hurt either way, done and tested.